liberty-maven-plugin/src/main/java/io/openliberty/tools/maven/server/StartDebugMojoSupport.java

```diff
@@ -462,7 +462,7 @@ private void copyDependencies(Dependency dep, String overrideLocation, String de
                 if (stripVersion) {
                     targetFileName = stripVersionFromName(targetFileName, nextArtifact.getVersion());
                 }
-           
+
```

Comment:
Now that we are taking a `classifier`, do we need to provide `stripClassifier` similar to `stripVersion`?

Reply:
Even worse, we should make sure (using a unit test)9 that `stripVersionFromName` will still work correctly!

As for the `stripClassifier`: It is uncommon to include a dependency with multiple versions (into the same directory). The same cannot be said about artifacts with classifiers. For this reason, we should not implement this.

That said, looking into the future, `getFile().getName()` and then operating on it might not be a wise choice: Johnzon's jakarta dependency is `apache-johnzon-1.20.4-jakarta.jar` and you could also have artifacts like `my-webapp-1.0.0-static.tar.gz` and `my-webapp-1.0.0-properties.tar.gz` next to `my-webapp-1.0.0.war`. Stripping the qualifiers would make you loose a file.

Therefore, if you want to have this option in this commit, a rewrite to manually concatinating the gid-artifactid-version-classifier.type would be feasible here.

Reply:
And this is why we didn't implement this small quick fix when the original issue was opened. There are other factors to consider, more design to think of. Unfortunately I cannot spend more time on this right now as I have other pressing deadlines.

I will look at your additions/changes as time permits. Thank you for spending time to add doc and tests.

Reply:
> There are other factors to consider, more design to think of. 

Yes, I find the current design fragile. You are currently relying on the file naming of the file in the maven Repository. Instead, if you added the components manually (e.g. join artifactId, version, type) you would not need a complex stripVersion method and had a stable method.